cli/cli.go

```diff
@@ -458,6 +461,50 @@ func getCreateInitialMigrationSubcommand() cli.Command {
 	}
 }
 
+func getGenerateGoextCommand() cli.Command {
```

Comment:
maybe this should be renamed to getConverter command

Reply:
this name is already used by different command

Reply:
I've change the name to getConverterCommand